src/azure-cli-testsdk/azure/cli/testsdk/checkers.py

```diff
@@ -39,6 +39,24 @@ def __call__(self, execution_result):
                                               execution_result.output)
 
 
+class JMESPathCheckGreaterThan(object):  # pylint: disable=too-few-public-methods
```

Comment:
@jaredmoo does this help? http://jmespath.org/proposals/filter-expressions.html?highlight=greater%20than#comparison-operators